test/hermes_test/commands/init/test_init.py

```diff
@@ -1,10 +1,12 @@
-# SPDX-FileCopyrightText: 2024 Forschungszentrum Jülich
+# SPDX-FileCopyrightText: 2024 Forschungszentrum Jülich GmbH
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileContributor: Nitai Heeb
 
```

Comment:
Regarding the tests executed here:

- IMHO we should discuss introducing integration testing using pytest + capsys / pexpect for the most happy paths. At least some very basic stuff - like calling the command without getting exceptions etc.

- Using pexpect or capsys the slim_click implementation can also be unit tested.

- The OAuth handler for interactive browser logins should be able to test with an integration test using something like Testcontainers to create a toy IdP and using Playwright for the browser remote control. Maybe this is a task left for later, but it would be good to have a test for this piece.